plugins/inputs/ethtool/namespace_linux.go

```diff
@@ -94,7 +94,7 @@ func (n *NamespaceGoroutine) Start() error {
 		}
 		if !initialNamespace.Equal(n.handle) {
 			if err := netns.Set(n.handle); err != nil {
-				n.Log.Errorf(`Could not switch to namespace "%s": %s`, n.name, err)
+				n.Log.Errorf("Could not switch to namespace %q: %w", n.name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.